catroid/src/test/java/org/catrobat/catroid/test/physics/clone/CloneWhenBounceOffBrick.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.test.physics.clone;
+
+import org.catrobat.catroid.content.WhenBounceOffScript;
+import org.catrobat.catroid.content.bricks.Brick;
+import org.catrobat.catroid.physics.content.bricks.WhenBounceOffBrick;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import static junit.framework.Assert.assertNotSame;
+
+@RunWith(JUnit4.class)
+public class CloneWhenBounceOffBrick {
+	@Test
+	public void testCloneWhenBounceOffBrick() throws Exception {
+		Brick brick = new WhenBounceOffBrick(new WhenBounceOffScript(null));
+		Brick clonedBrick = brick.clone();
+
+		WhenBounceOffScript brickReceiverScript = (WhenBounceOffScript) brick.getScript();
+		WhenBounceOffScript clonedBrickReceiverScript = (WhenBounceOffScript) clonedBrick.getScript();
+
+		assertNotSame(brickReceiverScript, clonedBrickReceiverScript);
+	}
+}
```

Comment:
This test can be deleted see [CloneScriptBrickTest](https://github.com/Catrobat/Catroid/blob/develop/catroid/src/test/java/org/catrobat/catroid/test/content/bricks/CloneScriptBrickTest.java)